doc/v3_doc.yaml

```diff
@@ -390,6 +390,13 @@ compute_parameters:
         # of diffusive network segments, organized by tailwater ID (keys).
         # (!!) mandatory for hybrid simulations
         diffusive_domain:
+        # ---------------
+        # filepath to topobathy data for channel cross sections. 
+        # This file is currently only NetCDF file format and contain  
+        # topobathy data of a channel cross section defined by comid.
+        # Theoretically, comid is identical to link of RouteLink data.
+        # (!!) mandatory for diffusive routing for natural cross sections.
+        topobathy_domain:
```

Comment:
Please add configuration documentation for `run_hybrid_routing` and `use_natl_xsections`. These are boolean variables that indicate whether or not hybrid routing and natural cross sections are to be included in a simulation. Their values default to `False`, so setting them to `True` is mandatory for hybrid and natural cross section capabilities. Additionally, when `use_natl_xsections` is `False`, then trapezoidal geometries from the RouteLink file are used